### PR TITLE
Add Expose service

### PR DESCRIPTION
**Pull request by chrisdicarlo**
Add Expose server support.

Adds a new category "Tools" to the `enable` menu.

Requires some external setup to get fully working:

- Wildcard DNS entry for Expose domain
- Reverse proxy or port forward to host from which takeout is running

Additional info in issue #166

### CODE REVIEW

File: app/Services/Expose.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            'prompt' => 'What is the username?',
+            'default' => 'username',
+        ],
+        [
+            'shortname' => 'password',
+            'prompt' => 'What is the password?',
+            'default' => 'password',
+        ],
+    ];
+
```
Comment (mattstauffer) [inline app/Services/Expose.php:34]: Delete extra line please

**Comment by mattstauffer** [comment]
Looks good just with one extra line deleted!

**Comment by chrisdicarlo** [comment]
Whoops, screwed up the rebase.

**Comment by mattstauffer** [comment]
Thanks @chrisdicarlo!

**Pull request merged**